navigation/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/NavigationConstants.java

```diff
@@ -13,8 +13,6 @@
   @Experimental public static final int MEDIUM_ALERT_LEVEL = 3;
   @Experimental public static final int HIGH_ALERT_LEVEL = 4;
   @Experimental public static final int ARRIVE_ALERT_LEVEL = 5;
-  @Experimental public static final int METERS_TO_INTERSECTION = 50;
-  @Experimental public static final int DEFAULT_ANGLE_TOLERANCE = 45;
```

Comment:
They were part of previous logic I implemented that I removed in favor of what iOS has, these constants just never got removed and noticed while I was working on this PR.